resources/assets/js/annotations/stores/settings.js

```diff
@@ -9,6 +9,7 @@ import Settings from '../../core/models/Settings';
 let defaults = {
     // Settings tab.
     annotationOpacity: 1.0,
+    cachedImages: 1,
```

Comment:
This needed to be added, too. I forgot about that myself. It's all quite redundant...